auctions/views.py

```diff
@@ -116,7 +116,7 @@ def post(self, request, *args, **kwargs):
                 if claim.payout_request():
                     messages.success(request, 'Your payout was sent.')
                 else:
-                    messages.error(request, "Your payout has been requested.")
+                    messages.error(request, "Sorry please try later")
```

Comment:
This will fail if the user doesn't have a Stripe account. Can we show them a message like "You need a Stripe account to request payout."?